src/components/Garden/ProposalDetail/ProposalStatus.tsx

```diff
@@ -57,21 +75,25 @@ export function getStatusAttributes(proposal, theme) {
   }
 }
 
-const ProposalStatus = ({ proposal }) => {
+type ProposalStatusProps = {
+  proposal: ProposalType
+}
+
+const ProposalStatus = ({ proposal }: ProposalStatusProps) => {
   const theme = useTheme()
 
-  const { Icon, iconSrc, color, label } = getStatusAttributes(proposal, theme)
+  const attributes = getStatusAttributes(proposal, theme)
 
   return (
     <Main
       css={`
         ${textStyle('body2')};
-        color: ${color || theme.surfaceContentSecondary};
+        color: ${attributes?.color || theme.surfaceContentSecondary};
       `}
     >
-      {iconSrc ? (
+      {attributes?.iconSrc ? (
         <img
-          src={iconSrc}
+          src={attributes?.iconSrc}
```

Comment:
I think question mark (?) its not necessary here.

Reply:
Yep.

Reply:
Fixed 👌

Reply:
Yah I agree.